blocks/gu_spdetails/block_gu_spdetails.php

```diff
@@ -333,8 +329,9 @@ public static function return_gradingduedate($finalgrade, $gradingduedate) {
         $duedateobj = new stdClass();
         $duedateobj->hasgrade = (!empty($finalgrade)) ? true : false;
         $duedateobj->gradetext = (!empty($finalgrade)) ? $finalgrade :
-                                get_string('due', SPDETAILS_STRINGS).
-                                userdate($gradingduedate, get_string('convertdate', SPDETAILS_STRINGS));
+                                 ((time() > $gradingduedate) ? ucwords(get_string('overdue', SPDETAILS_STRINGS)) :
```

Comment:
Needs to consider the scenario for 'To be confirmed'.

Reply:
Fixed at commit [7d7091b](https://github.com/uofg-andf/andfmoodle/pull/184/commits/7d7091baa7b2782798a933e84e30c7374545da56)